src/main/java/com/sleekbyte/tailor/output/Printer.java

```diff
@@ -108,6 +108,18 @@ public void ignoreLine(int ignoredLineNumber) {
         this.ignoredLineNumbers.add(ignoredLineNumber);
     }
 
+    /**
+     * Suppress analysis output for a given region.
+     *
+     * @param start line number where the region begins
+     * @param end line number where the region ends
+     */
+    public void ignoreRegion(int start, int end) {
+        for (int i = start; i <= end; i++) {
+            this.ignoredLineNumbers.add(i);
```

Comment:
I think we should store `ignoredRegions` instead of `ignoredLineNumbers`. Single lines that need to be ignored can be stored as `<LineNum, LineNum>`.